script-analysis/summarize.mjs

```diff
@@ -78,7 +90,29 @@ for (let i = 0; i < json.length; i++) {
         // this should never happen either.
         unified[unifiedTag] = { n, wrapperMicros: totalMicros, wrapperMean: mean };
       }
+    } else if (tag.endsWith(post)) {
+      const unifiedTag = tag.slice(prefixLen, -postLen);
+
+      unified[unifiedTag] = { n, wrapperMicros: totalMicros, wrapperMean: mean };
+      unified[unifiedTag].tag = unifiedTag + ' post';
+
+      summarized.push(unified[unifiedTag]);
+    } else if (tag.endsWith(put)) {
+      const unifiedTag = tag.slice(prefixLen, -putLen);
+
+      unified[unifiedTag] = { n, wrapperMicros: totalMicros, wrapperMean: mean };
+      unified[unifiedTag].tag = unifiedTag + ' put';
+
+      summarized.push(unified[unifiedTag]);
+    } else if (tag.endsWith(get)) {
+      const unifiedTag = tag.slice(prefixLen, -getLen);
+
+      unified[unifiedTag] = { n, wrapperMicros: totalMicros, wrapperMean: mean };
+      unified[unifiedTag].tag = unifiedTag + ' get';
+
+      summarized.push(unified[unifiedTag]);
```

Comment:
Probably could clean this up since it's a lot of repeat code around the tags but since it's for testing purposes only it doesn't matter much